src/GovUk.Education.ExploreEducationStatistics.Content.Api/Startup.cs

```diff
@@ -246,15 +246,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
         app.UseRewriter(rewriteOptions);
 
         app.UseCors(options =>
-            options
-                .WithOrigins(
-                    "http://localhost:3000",
-                    "http://localhost:3001",
-                    "https://localhost:3000",
-                    "https://localhost:3001"
-                )
-                .AllowAnyMethod()
-                .AllowAnyHeader()
+            options.WithOrigins("http://localhost:3000", "https://localhost:3000").AllowAnyMethod().AllowAnyHeader()
```

Comment:
Should this be https://localhost:3001?

Same comment applies to the two other places.